robyn/openapi.py

```diff
@@ -241,11 +241,7 @@ def get_path_obj(self, endpoint: str, summary: str, tags: List[str], query_param
             "summary": summary,
             "tags": tags,
             "parameters": openapi_parameter_object,
-            "responses": {
-                "200": {
-                    "description": "Successful Response",
-                }
-            },
+            "responses": {"200": {"description": "Successful Response", "content": {"application/json": {"schema": {}}}}},
```

Comment:
@VishnuSanal , we have typing at the moment. e.g. regular response vs json response. We need to distinguish between that on return type.